### PR TITLE
Add TM1637 7-segment led display library

### PR DESCRIPTION
**Pull request by kmihaylov**
This library can be included without any patches. Works out of the box.

### CODE REVIEW

**Comment by slaff** [comment]
@kmihaylov Can you please record a short video demoing the working display?

File: .gitmodules [hunk 1]
```diff
@@ -85,3 +85,6 @@
 	path = Sming/Libraries/DFRobotDFPlayerMini
 	url = https://github.com/DFRobot/DFRobotDFPlayerMini.git
 	ignore = dirty
+[submodule "Sming/Libraries/TM1637"]
+	path = Sming/Libraries/TM1637
+	url = https://github.com/avishorp/TM1637.git
```
Comment (slaff) [inline .gitmodules]: Add also the following line at the end:
```
          ignore = dirty
```

File: samples/Display_TM1637/app/application.cpp [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+	for(uint8_t z = 0; z < 2; z++) {
+		for(k = 0; k < 10000; k += k * 4 + 7) {
+			display.showNumberDec(k, lz);
+			delay(TEST_DELAY);
+		}
+		lz = true;
+	}
+
+	// Show decimal number whose length is smaller than 4
+	for(k = 0; k < 4; k++)
```
Comment (slaff) [inline samples/Display_TM1637/app/application.cpp]: Please, add opening and closing curly braces around the for-loop.

File: samples/Display_TM1637/app/application.cpp [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+	delay(TEST_DELAY);
+	display.showNumberDec(0, true, 1, 2);
+	delay(TEST_DELAY);
+	display.showNumberDec(0, true, 1, 1);
+	delay(TEST_DELAY);
+	display.showNumberDec(0, true, 1, 0);
+	delay(TEST_DELAY);
+
+	// Brightness Test
+	for(k = 0; k < 4; k++)
```
Comment (slaff) [inline samples/Display_TM1637/app/application.cpp]: Curly braces around for-loop.

**Comment by kmihaylov** [comment]
Here is the video https://youtu.be/Y-nCdiOydXo

**Comment by slaff** [comment]
> Here is the video https://youtu.be/Y-nCdiOydXo

Perfect, thanks!

**Pull request merged**